packages/synapse-react-client/src/components/GenericCard/CollapsibleDescription.tsx

```diff
@@ -34,8 +34,11 @@ export function LongDescription(props: {
     descriptionSubTitle,
     descriptionConfig,
   } = props
-  let content: JSX.Element | string = description
-  if (descriptionConfig?.isMarkdown) {
+  let content: ReactNode = description
+  if (
+    descriptionConfig?.isMarkdown &&
+    (content == null || typeof content === 'string')
```

Comment:
Add typeguard, does not change behavior (this would have crashed if `content` was not null or a string)